packages/nimble-components/src/table-column/anchor/cell-view/template.ts

```diff
@@ -0,0 +1,47 @@
+import { html, ref, when } from '@microsoft/fast-element';
+import type { TableColumnAnchorCellView } from '.';
+import { anchorTag } from '../../../anchor';
+
+export const template = html<TableColumnAnchorCellView>`
+    ${when(
+        x => typeof x.cellRecord.href === 'string',
+        html<TableColumnAnchorCellView>` <${anchorTag}
+            ${ref('anchor')}
+            href="${x => x.cellRecord.href}"
+            hreflang="${x => x.columnConfig.hreflang}"
+            ping="${x => x.columnConfig.ping}"
+            referrerpolicy="${x => x.columnConfig.referrerpolicy}"
+            rel="${x => x.columnConfig.rel}"
+            target="${x => x.columnConfig.target}"
+            type="${x => x.columnConfig.type}"
+            download="${x => x.columnConfig.download}"
+            underline-hidden
```

Comment:
I can't recall if we discussed this in the HLD: do we plan to support the `appearance` attribute from `nimble-anchor` on the anchor column too? My inclination would be to support it unless designers have guidance explaining why we don't want it.

If we decide to support it we should add tests and storybook documentation for it too.

Reply:
I didn't call it out in the HLD, and I'd just assumed we didn't need to support it. But I've gone ahead and added it, since there's no obvious reason not to.

Reply:
Did we get confirmation from Rick / Brandon what modes we want to expose in the table? This is hard coded to underline-hidden but the table spec on column alignments [shows underlines on the hyperlinks](https://xd.adobe.com/view/5b476816-dad1-4671-b20a-efe796631c72-0e14/screen/656fa89e-8005-4b8d-bd8e-cbbe7a6cafd4/specs/)

![image](https://user-images.githubusercontent.com/1588923/234415434-af47a394-b33f-438c-b418-528b56768445.png)

That looks like the prominent appearance with the underline visible. Do we need to be able to express all 4 permutations?

Reply:
Related, we have a visual style for disabled links: 
![image](https://user-images.githubusercontent.com/1588923/234415969-bd21b7d4-1cd3-4f51-b2bf-6ab374af4fdc.png)

Should the no link case look like a disabled link instead of plain text?

![image](https://user-images.githubusercontent.com/1588923/234416094-3d3d9963-6536-48ef-bb44-5a07c01ac708.png)

Reply:
![image](https://user-images.githubusercontent.com/7282195/234682885-14bc39b7-2b6f-481b-931e-e78d2d0ddd0d.png)
So it sounds like we want support for underlined links in the table. But should that be the default? SLE's table links only have underlines on hover.